packages/block-editor/src/hooks/layout.js

```diff
@@ -53,8 +44,13 @@ function LayoutPanel( { setAttributes, attributes, name: blockName } ) {
 		return null;
 	}
 
-	const allowLayoutSwitching = canBlockSwitchLayout( blockName );
-	const { inherit = false, type = 'default' } = layout;
+	const {
+		allowSwitching: canBlockSwitchLayout,
+		default: defaultBlockLayout,
+	} = getBlockSupport( blockName, layoutBlockSupportKey ) || {};
```

Comment:
I think we should have a flag to disable all layout UI: Basically you should be able to choose a default layout but do not allow changes by the user.

Reply:
I add an `allowEditing = true` flag and one more `allowInheriting` for displaying the respective control. Seems like many flags are needed.. Do you have any thoughts if we can simplify these?

Reply:
I'm not sure yet but yes, it's something we need to think about while it's still experimental.